src/Library/demos/Calendar/main.js

```diff
@@ -0,0 +1,27 @@
+const calendar = workbench.builder.get_object("calendar");
+
+function handleChange() {
+  let year = calendar.year;
+  let month = calendar.month;
+  let day = calendar.day;
+  let monthString = month.toString().padStart(2, "0");
+  let dayString = day.toString().padStart(2, "0");
+
+  console.log(
+    `Selected Date (YYYY-MM-DD): ${year}-${monthString}-${dayString}`,
+  );
```

Comment:
please use `getDate` instead - it's the perfect opportunity to showcase the DateTime API :)

https://docs.gtk.org/glib/method.DateTime.format.html

Please use this to display the date:

> \%x: the preferred date representation for the current locale without the time

Reply:
sure